index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class='head'> 
+            <div class="name">Yohannes Zemed </div>
+           <img src="icons/menu.png" alt='menu'/>
+        </div>
+    </header>
+    <section class="heading">
+
+         
+      <div class="main-text">
+        <div class ='text'>
+         <div>Hey there.</div>
+         <div >I'm Yohannes</div>
+        </div>
+        <div class ='text1'>I'am a software Developer</div>
+        <div class ='text2'>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</div>
+       </div>
+       <div class='contacts'>
+           <img src="icons/github.svg" alt="github addres"/>
+           <img src="icons/angelist.png" alt='angelist addres'/>
+           <img src="icons/medium.png"  alt='medium addres'/>
+           <img src="icons/Linkedin icon.svg" alt="linkedin addres"/>
+           <img src="icons/twitter.png" alt="twitter address"/>
+
+       </div>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)